bundles/org.openhab.binding.miio/src/main/resources/misc/device_names.json

```diff
@@ -0,0 +1,2725 @@
+  "090615.switch.mets1": "PTX one key intelligent switch (mesh)",
+  "090615.switch.mets2": "PTX two key intelligent switch (mesh)",
+  "090615.switch.mets3": "PTX three key intelligent switch (mesh)",
+  "090615.switch.mhrsw1": "PTX Sle Bluetooth intelligent switch moduleing",
+  "090615.switch.mhrsw2": "PTX dual Bluetooth intelligent switch module",
+  "090615.switch.piak01": "Piak intelligent one switch",
+  "090615.switch.piak02": "Piak intelligent two switch",
+  "090615.switch.piak03": "Piak intelligent three switch",
+  "090615.switch.switch01": "One Intelligent switch(WIFI)",
+  "090615.switch.switch02": "Two Intelligent Switch (WIFI)",
+  "090615.switch.switch03": "Three Intelligent switch(WIFI)",
+  "090615.switch.xswitch01": "PTX OneKey Switch (WIFI)",
+  "090615.switch.xswitch02": "PTX Twokey switch(wifi)",
+  "090615.switch.xswitch03": "PTX ThreeKey Switch (WIFI)",
+  "123888.aircondition.z": "aircondition",
+  "123888.fan.1121": "电风扇",
+  "123888.plug.znc102": "智能插座",
+  "123888.stb.1101": "机顶盒",
+  "123888.tv.xhzn11": "电视",
+  "1245.airfresh.mini2": "Potato Mini",
+  "1245.airpurifier.dl01": "DATUDOU SUPER",
+  "17216.desk.lt001": "Leband Electric Sit-to-Stand Desk",
+  "17216.magic_touch.d150": "MagicTouch Warm",
+  "17216.magic_touch.d152": "MagicTouch-Go",
+  "17216.massage.ec1266a": "Easewell AI Massage Chair",
+  "17216.massage.j6261a": "joypal AI Health Master",
+  "17216.massage.jp3209": "joypal Health Smart",
+  "180.plug.plug0001": "Ai-Thinker Smart Socket",
+  "360sdj.vacuum.s": "360AICleanRobotics",
+  "364656.light.sm01a": "科技生活",
+  "397.light.hallight": "Stars LED Light",
+  "666.curtain.em75": "A-OK Curtain Controller",
+  "666.curtain.gt01": "Smart Curtain Motor",
+  "666.curtain.gt01m": "Genwits Smart Curtain",
+  "666.curtain.id92": "WIFI Smart Curtain",
+  "693948.aircondition.khp": "KH Air Controller",
+  "693948.curtain.khcl": "KH Curtain",
+  "693948.plug.khgsp": "KH POWER PULG",
+  "693948.stb.khp": "KH set-top box",
+  "693948.switch.khkz": "KH Zero Fire Switch",
+  "693948.tv.khp": "KH TV Controller",
+  "759413.aircondition.iez": "日立中央空调",
+  "881878.curtain.clznljq": "Smart Module for Electric Curtain",
+  "881878.light.ddznljq": "Smart Module for LED Strip",
+  "881878.light.znxyd": "Smart Night Light",
+  "881878.switch.ddkgznljq": "Smart Module for Electric Switch",
+  "a24234.plug.89987": "创建云接入",
+  "aden.airc.a6": "VINO Inverter Air Conditioner 3P(New Energy Label Level 2)",
+  "aden.aircondition.a1": "VINO Inverter Air Conditioner(Energy Label Level 3)",
+  "aden.aircondition.a2": "VINO Inverter Air Conditioner(Energy Label Level 1)",
+  "aden.aircondition.a4": "VINO Inverter Air Conditioner",
+  "afour.s_lamp.001": "ZIWOOO Intelligent germicidal lamp",
+  "aice.motor.kzmu3": "KMZ intelligent shutter door controller",
+  "aihome.light.m85": "light",
+  "aihome.light.m93": "FanLight",
+  "aiicn.curtain.r51": "智能窗帘控制器",
+  "aiicn.light.pwm2": "LED驱动器",
+  "aiicn.light.r01": "单路接收器",
```

Comment:
I'm not sure that chinese names would really help english speakers know what they are looking at...

Reply:
I'm sure not... these devices don't have an English description in the cloud, guess they are only for sale in CN.
This list is all devices that are known for Xiaomi, that does not mean they are included in OH..

Dunno an easy fix for this (I can delete all non-english ones, but that does not solve anything)

Reply:
I'm not really sure there is a good fix either. But I think as long as you only show the device name as additional information for the device and not as the key identifier for the device, both in openHAB and in documentation, then I think we should be ok.

Reply:
When I actually implement such a device I normally use the English names. Until very recent I didn't even know what is the name of a device according to Xiaomi. I used a mix of commercial names ( how they call the devices on the webshops) or generic names based on the model e.g aiicn.light.r01 would become aiicn light, and it still unclear Google translate as final solution

Long story short... When I actually add a device, I'll add it with an English name.
The inclusion here in the readme maker is just to speedup/simplify the creation of the actual Java devicelist file.